tests/integration/asyncio/connection_strategy_test.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+
+from hazelcast.asyncio import HazelcastClient
+from hazelcast.config import ReconnectMode
+from hazelcast.errors import ClientOfflineError, HazelcastClientNotActiveError
+from hazelcast.lifecycle import LifecycleState
+from tests.integration.asyncio.base import HazelcastTestCase
+from tests.util import random_string
+
+
+class ConnectionStrategyTest(unittest.IsolatedAsyncioTestCase, HazelcastTestCase):
```

Comment:
Why did you remove async start tests? I think these can be used on async client without awating to see client's reaction.